src/index.ts

```diff
@@ -1,22 +1,27 @@
-import "./index.css"
+// import * as uint8arrays from "uint8arrays"
```

Comment:
I commented out all the code in this file that's not being used right now because some of the comments outline the next steps for the app once we have everything working end-to-end. Maybe I should just remove all the commented code for now though since we have it in git? 🤷🏼

Reply:
All good to remove yeah 👍

Reply:
> All good to remove yeah 👍

will do!